mmdet3d/ops/furthest_point_sample/src/furthest_point_sample_cuda.cu

```diff
@@ -207,3 +207,192 @@ void furthest_point_sampling_kernel_launcher(int b, int n, int m,
     exit(-1);
   }
 }
+
+template <unsigned int block_size>
+__global__ void furthest_point_sampling_with_dist_kernel(
```

Comment:
`furthest_point_sampling_with_dist_kernel` -> `furthest_point_sampling_by_dist_kernel`

Reply:
The name of the function means computing furthest points sampling **with distance inputs**.